lib/php-saml/CHANGELOG

```diff
@@ -0,0 +1,59 @@
+CHANGELOG
```

Comment:
Why version 2.4.0 which is 1 year old ?
Upstream have released 2.50, 2.6.0, 2.6.1, 2.7.0.